src/components/Splash/Splash.scss

```diff
@@ -0,0 +1,16 @@
+@import '~theme/variables.scss';
+
+:export {
+  brandPrimary: $brand-primary;
```

Comment:
@ericgold very cool - I did not know you could do this!

Reply:
me neither! I copied it from somewhere else in the app...I think a different place where we already had a loader?